### DISCUSSION TITLE
Traceability of resources from Cloud to Code

### DISCUSSION THREAD

**Discussion by fatihtokus** [aquasecurity/trivy#4127] [Q&A] [answered]
Hello all,

I am looking for a mechanism to trace resources from the cloud console to the code(repo). In this way, we will be able to audit the cloud changes from git. And we will say that this resource was approved via this PR and reviewed by that person.

Does trivy have a feature like that?

Many thanks,
Fatih

**Comment by knqyf263** [comment]
Sounds like you're looking for IaC. Something else?

**Comment by fatihtokus** [comment]
Yes, we are using terraform. But it doesn't come with Cloud to Code traceability, or does it?

**Reply by itaysk** [reply, marked as answer]
what you are looking for doesn't sound related to Trivy, which is a security scanner.

**Reply by fatihtokus** [reply]
I think you are right @itaysk. After rethinking it, it is not part of trivy. But it is something that completes the end2end IAC pipeline. 
By the way, I found a very nice open source tool that does what I was looking for. https://yor.io/
